packages/clerk-js/src/ui/elements/PasswordInput.tsx

```diff
@@ -7,15 +9,20 @@ import { usePassword } from '../hooks/usePassword';
 import { Eye, EyeSlash } from '../icons';
 import { useFormControl } from '../primitives/hooks';
 import type { PropsOfComponent } from '../styledSystem';
+import { mergeRefs } from '../utils';
 import { IconButton } from './IconButton';
 
 type PasswordInputProps = PropsOfComponent<typeof Input> & {
   validatePassword?: boolean;
 };
 
+const DEBOUNCE_MS = 350;
```

Comment:
Is is a const we are using in more that one file, could we place it an a util file or a "shared" file ?

Reply:
I'm moving this elsewhere because I decided to reuse it.